RoddyCore/src/de/dkfz/roddy/knowledge/methods/GenericMethod.groovy

```diff
@@ -78,6 +78,15 @@ class GenericMethod {
     }
 
     /**
+     * If you need a file group output and the files are already set in the cfg, then this is the right method to call.
```

Comment:
What has this method to do with the config? The configs are not part of the interface, so this is not obvious.